tests/e2e/utils/flows.js

```diff
@@ -203,6 +203,10 @@ export const shopperWCP = {
 			'#billing-postcode',
 			customerBillingDetails.postcode
 		);
+		await clearAndFillInput(
+			'#billing-state input',
```

Comment:
Do you think it makes sense to add a direct identifier for this element instead of accessing the `input` through another selector? Might be more explicit like the other ones we are using to clear inputs.

Reply:
Looks like this isn't relevant to the fix. Was it relevant before? Did I get something wrong? Dunno, but I've removed it.